source/includes/_ob_errors.md

```diff
@@ -0,0 +1,74 @@
+# Open Banking Errors
+
+Our open banking APIs return errors inline with the [open banking specification](https://openbankinguk.github.io/read-write-api-site3/v3.1.10/profiles/read-write-data-api-profile.html#error-response-structure).
+
+## OBErrorResponse1
+
+| Name    | Type         | Description                                              |
+| ------- | ------------ | -------------------------------------------------------- |
+| Code    | Max40Text    | Custom error code with a format of `errPrefix`.`errType` |
+| Id      | Max40Text    | Internal trace identifier                                |
+| Message | Max500Text   | High level human readable category message               |
+| Errors  | `[]OBError1` | One element array containing both a `Code` & `Message`   |
```

Comment:
> One element array containing both a `Code` & `Message`

this isn't super clear. Sounds like it'll be `[Code, Message]`. I'd add the relevant rows for `OBError1.Code` and `OBError1.Message`